docker-compose.yaml

```diff
@@ -75,4 +78,33 @@ services:
       timeout: 10s
       retries: 5
       start_period: 40s
-    command: mongod --quiet --logpath /dev/null
\ No newline at end of file
+    command: mongod --quiet --logpath /dev/null
+
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 8004
+      ZOOKEEPER_TICK_TIME: 2000
+
+  kafka:
+    image: confluentinc/cp-kafka:latest
+    hostname: kafka
+    depends_on:
+      - zookeeper
+    environment:
+      KAFKA_BROKER_ID: 1
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:8004
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:8005,PLAINTEXT_HOST://localhost:28005
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+
+  kafka-ui:
+    container_name: kafka-ui
+    image: provectuslabs/kafka-ui:latest
+    ports:
+      - "8006:8080"
+    environment:
+      DYNAMIC_CONFIG_ENABLED: 'true'
+    volumes:
+      - ./kui/config.yaml:/etc/kafkaui/dynamic_config.yaml
```

Comment:
Would it be better to change this to `./backend/matching-service/kui/config.yaml:/etc/kafkaui/dynamic_config.yaml` and move the `kui` folder to inside backend/matching-service

Reply:
Good point, it's not under the matching service though so I'll shift it into backend folder instead